tests/test_rdataframe.py

```diff
@@ -0,0 +1,47 @@
+import unittest, ROOT
+from PyRDF import RDataFrame, RDataFrameException
+
+class RDataFrameConstructorTests(unittest.TestCase):
+
+    def test_integer_arg(self):
```

Comment:
Maybe you could add another commit to this PR with the corresponding docs for this test file?

Reply:
Sure @etejedor